src/components/Description/index.tsx

```diff
@@ -21,30 +37,52 @@ class Description extends PureComponent<Props> {
   render() {
     const { textBlocks, literature, withoutIDs } = this.props;
     const sections = textBlocks.map((e) => {
-      return transformFormatted(e) as string[];
+      return (transformFormatted(getDescriptionText(e)) as string[]).map(
+        (text) => ({
+          text,
+          llm: typeof e === 'string' ? false : e.llm,
+          checked: typeof e === 'string' ? false : e.checked,
+        }),
+      );
     });
 
     return (
       <div className={css('description')} data-testid="description">
         {sections
           .map((section, i) =>
             section
-              .map((p, j) => (
-                <Paragraph
-                  key={`${i}.${j}`}
-                  p={p}
-                  literature={literature || []}
-                  withoutIDs={withoutIDs}
-                />
+              .map(({ text, llm, checked }, j) => (
+                <div className={css({ llm, checked })} key={`${i}_${j}`}>
+                  {llm && (
+                    <Tooltip title="Paragraph generated by GPT-4">
+                      <span
+                        className={css('icon', 'icon-common', 'icon-magic')}
+                      />
+                    </Tooltip>
+                  )}{' '}
+                  {checked && (
+                    <Tooltip title="A curator has checked the content of this generated paragraph">
+                      <span
+                        className={css('icon', 'icon-common', 'icon-check')}
+                      />
+                    </Tooltip>
```

Comment:
Showing the source of the description block using badges instead of icons:

```suggestion
                <div className={css('content', { llm, checked })} key={`${i}_${j}`}>
                  {llm ? (
                    <span className={css('vf-badge', 'vf-badge--tertiary')}>
                      AI-generated
                      <span className={css('details')}>{checked ? 'Reviewed' : 'Unreviewed'}</span>
                    </span>
                  ) : (
                    <span className={css('vf-badge', 'vf-badge--tertiary')}>
                      Expert-curated
                    </span>
```

Instead of using a colored background, I'd suggest a colored left border when the description is AI-generated _and_ unreviewed (see changes to `Description/styles.css`).